spec/models/team_performance_spec.rb

```diff
@@ -186,14 +141,6 @@ def create_all_teams
         end
       end
 
-      context 'for a team with older activities and older feedback' do
-        subject { TeamPerformance.new(team_both_outdated)}
-
-        it "signals green" do
-          expect(subject.evaluation).to eq(:green)
-        end
-      end
-
```

Comment:
This one, I am pretty sure it _should_ stay, because it is to ensure that outside of the season, the teams' performance is frozen to green.
I also begin to suspect that `team_both_outdated` is the same as either `team_nothing` or `team_both`. Which is a nice illustration that naming is hard :-) But hey, it was my first year in Ruby ;-)